tests/NuGet.Services.BasicSearchTests/Fixtures/StartupTestsFixture.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using NuGet.Services.BasicSearch;
+using NuGet.Services.BasicSearchTests.TestSupport;
+
+namespace NuGet.Services.BasicSearchTests.Fixtures
+{
+    public class StartupTestsFixture : IDisposable
+    {
+        private readonly Lazy<StartedWebApp<Startup>> _lazyApp;
+
+        public StartedWebApp<Startup> App => _lazyApp.Value;
+
+        public StartupTestsFixture()
+        {
+            _lazyApp = new Lazy<StartedWebApp<Startup>>(() => InitializeAsync().Result);
+        }
+
+        public void Dispose()
+        {
+            if (_lazyApp.IsValueCreated)
+            {
+                _lazyApp.Value.Dispose();
+            }
+        }
+
+        private async Task<StartedWebApp<Startup>> InitializeAsync()
+        {
+            // list packages that each test needs
+            var packageVersions = new Dictionary<string, IEnumerable<PackageVersionIdentifier>>
+            {
+                {
+                    nameof(StartupTests.AutocompleteParameterIsObserved),
+                    new[]
+                    {
+                        new PackageVersionIdentifier("Antlr", "3.1.3.42154"),
+                        new PackageVersionIdentifier("Antlr", "3.4.1.9004-pre"),
+                        new PackageVersionIdentifier("angularjs", "1.2.0-RC1"),
+                        new PackageVersionIdentifier("WebGrease", "1.6.0")
+                    }
+                }
+            };
+
+            // construct the test settings
+            var settings = new BasicSearchTestSettings
```

Comment:
I've made the package list inline with the code and values the same for all tests in XML.